usaspending_api/awards/management/commands/load_subawards.py

```diff
@@ -0,0 +1,194 @@
+import logging
+import os
+
+from django.core.management.base import BaseCommand
+from django.db import transaction
+from functools import partial
+from psycopg2.sql import Identifier, Literal, SQL
+from usaspending_api.common.helpers.sql_helpers import convert_composable_query_to_string, get_connection
+from usaspending_api.common.helpers.timing_helper import Timer
+
+
+logger = logging.getLogger("console")
+
+
+# All of our timers log to logger.info and logger.error.
+Timer = partial(Timer, success_logger=logger.info, failure_logger=logger.error)
+
+
```

Comment:
Isn't it customary to put a comment here summarizing what this management command was created to do and when if ever it's obsolete?